### ISSUE TITLE
Vue extends not supported

### ISSUE THREAD

**Issue by freddy-daniel** [vue-styleguidist/vue-styleguidist#950]
**Not detecting documentation from extended js file**
```javascript
/**
 * Display QR Code image displayer
 * @displayName QRCode
 */
import qrcode from '@/extended/qrcodecontroller';
export default {
  name: 'QRCode',
  extends: qrcodecontroller,
  props: {
    /**
     * QR code image src
     */
    src: {
      type: String,
      default: null
    }
  }
};
```
Not detected event emit written in **qrcodecontroller** file

**Comment by elevatebart** [comment]
Is there a typo in your code?

```js
/**
 * Display QR Code image displayer
 * @displayName QRCode
 */
import qrcodecontroller from '@/extended/qrcodecontroller';
export default {
  name: 'QRCode',
  extends: qrcodecontroller,
  props: {
    /**
     * QR code image src
     */
    src: {
      type: String,
      default: null
    }
  }
};
```

**Comment by elevatebart** [comment]
Would you please share how your `qrcodecontroller` works? 

### 2 bits of explanation

When a component has the extends option, vue-docgen-api parses the extended component first as if it were a component. Once done, it merges both documentations, extended and local, into one, giving priority to the local one.

So if you want to test why this event is not detected in the extended component, it could be that the extended is not recognized as a component, or that the event triggered would not be parsed, even if it was the component targetted by vue-docgen-api.

Tell me if it helps.
Thank you in advance.
